apps/src/signUpFlow/FinishStudentAccount.tsx

```diff
@@ -139,123 +171,149 @@ const FinishStudentAccount: React.FunctionComponent<{
   };
 
   return (
-    <div className={style.finishAccountContainer}>
```

Comment:
I had to add another div here because we're adding a tos and privacy link outside this container!